src/test/java/com/hazelcast/commandline/member/MemberCommandLineTest.java

```diff
@@ -0,0 +1,193 @@
+package com.hazelcast.commandline.member;
+
+import com.hazelcast.commandline.CommandLineTestSupport;
+import com.hazelcast.commandline.HazelcastCommandLine;
+import com.hazelcast.core.LifecycleEvent;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.assertTrue;
+
+public class MemberCommandLineTest extends CommandLineTestSupport {
+
+    private MemberCommandLine memberCommandLine;
+
+    @Before
+    public void setup() {
+        resetOut();
+        memberCommandLine = new MemberCommandLine(out, err);
+    }
+
+    @After
+    public void close() throws IOException, InterruptedException {
+        killAllRunningInstances();
+        removeFiles();
+    }
+
+    private void removeFiles() throws IOException {
+        Path pathToBeDeleted = Files.createDirectories(Paths.get(HazelcastCommandLine.HAZELCAST_HOME));
+
+        Files.walk(pathToBeDeleted)
+                .sorted(Comparator.reverseOrder())
+                .map(Path::toFile)
+                .forEach(File::delete);
+    }
+
+    private void killAllRunningInstances() throws IOException, InterruptedException {
+        String out = getRunningProcesses();
```

Comment:
Won't kill this all java processes in the machine?

Reply:
No, the `getPids(out)` call in the next line specifies the pids of the running `HazelcastMember` instances. I might change the name of the method, for clarification.